apps/gp2-server/test/data-providers/working-group.data-provider.test.ts

```diff
@@ -107,68 +111,186 @@ describe('Working Group Data Provider', () => {
-      ${WorkingGroupsDataMembersRoleEnum.WorkingGroupMember} | ${'Working group member'}
-    `('should parse the role', ({ role, expectedRole }) => {
-      const workingGroup = getGraphQLWorkingGroup();
-      const member = getGraphQLWorkingGroupMember();
-      member!.role = role;
-      workingGroup.flatData.members = [member];
-      const { members } = parseWorkingGroupToDataObject(workingGroup);
-      expect(members[0]?.role).toEqual(expectedRole);
+      test('should skip the user from the result if the role property is undefined', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const member = getGraphQLWorkingGroupMember();
+        member!.role = null;
+        workingGroup.flatData.members = [member];
+        const { members } = parseWorkingGroupToDataObject(workingGroup);
+        expect(members).toEqual([]);
+      });
+      test.each`
+        role                                                   | expectedRole
+        ${WorkingGroupsDataMembersRoleEnum.Lead}               | ${'Lead'}
+        ${WorkingGroupsDataMembersRoleEnum.CoLead}             | ${'Co-lead'}
+        ${WorkingGroupsDataMembersRoleEnum.WorkingGroupMember} | ${'Working group member'}
+      `('should parse the role', ({ role, expectedRole }) => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const member = getGraphQLWorkingGroupMember();
+        member!.role = role;
+        workingGroup.flatData.members = [member];
+        const { members } = parseWorkingGroupToDataObject(workingGroup);
+        expect(members[0]?.role).toEqual(expectedRole);
+      });
+
+      test('should skip the user from the result if the user property is undefined', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const member = getGraphQLWorkingGroupMember();
+        member!.user = null;
+        workingGroup.flatData.members = [member];
+        const { members } = parseWorkingGroupToDataObject(workingGroup);
+        expect(members).toEqual([]);
+      });
     });
 
-    test('should skip the user from the result if the user property is undefined', () => {
-      const workingGroup = getGraphQLWorkingGroup();
-      const member = getGraphQLWorkingGroupMember();
-      member!.user = null;
-      workingGroup.flatData.members = [member];
-      const { members } = parseWorkingGroupToDataObject(workingGroup);
-      expect(members).toEqual([]);
+    describe('resources', () => {
+      test('should map a resource note', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const title = 'working group title';
+        const description = 'working group description';
+        const resource = getGraphQLWorkingGroupResource();
+        resource.type = WorkingGroupsDataResourcesTypeEnum.Note;
+        resource.title = title;
+        resource.description = description;
+        workingGroup.flatData.resources = [resource];
```

Comment:
have you considered having a fixture of a working group with a resource available by default?